mods/ra/rules/ships.yaml

```diff
@@ -255,10 +228,7 @@ LST:
 		MaxWeight: 5
 		PipCount: 5
 		PassengerFacing: 0
-	RepairableNear:
-	Explodes:
-		Weapon: UnitExplodeShip
-		EmptyWeapon: UnitExplodeShip
+	-Chronoshiftable:
```

Comment:
*shrugs* The landing craft didn't have this trait before I moved it to the ^Ship default, either, so this is no change vs bleed. This was probably done to work around balancing issues or "kill load" issues, but I don't know.